hedera-node/src/main/java/com/hedera/services/store/AccountStore.java

```diff
@@ -81,6 +84,33 @@ public Account loadAccount(Id id) {
 		return account;
 	}
 
+	/**
+	 * Returns a model of the requested token, with operations that can be used to
+	 * implement business logic in a transaction.
+	 *
+	 * <b>IMPORTANT:</b> Changes to the returned model are not automatically persisted
+	 * to state! The altered model must be passed to {@link ContractsState#persistAccount(com.hedera.services.store.models.Account)}
+	 * in order for its changes to be applied to the Swirlds state, and included in the
+	 * {@link com.hedera.services.state.submerkle.ExpirableTxnRecord} for the active transaction.
+	 *
+	 * @param id the account to load
+	 * @return a usable model of the account
+	 * @throws InvalidTransactionException if the requested contract is missing, deleted or is not smart contract
+	 */
+	public Account loadContract(Id id) {
+		final var key = new MerkleEntityId(id.getShard(), id.getRealm(), id.getNum());
+		final var merkleContracts = accounts.get().get(key);
+
+		validateUsable(merkleContracts);
+
+		final var account = new com.hedera.services.store.models.Account(id);
+		account.setExpiry(merkleContracts.getExpiry());
+		account.initBalance(merkleContracts.getBalance());
+		account.setAssociatedTokens(merkleContracts.tokens().getIds().copy());
```

Comment:
Is this needed?

Reply:
I moved it from ContractsStore and renamed ContractsStore to ContractsState after the call with @Daniel-K-Ivanov  yesterday.

Reply:
My question was releated to that specific line.